### PR TITLE
Failover

### PR DESCRIPTION
**Pull request by oat9002**
#4 Please take a look

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# How to edit a config
```
Comment (the-pat) [inline Readme.md]: Thanks for creating the README!

Could you change the readme to:
```md
# reddit-node-wallpaper

## Configuration
- `maxRetry`: the maximum number of times you want to retry
- `score`: the minimum score of the reddit post

```

File: index.js [hunk 2]
```diff
@@ -5,38 +5,48 @@ const request = require('request');
...
 const wallpaper = require('wallpaper');
 
-const change = () => {
-    const url = `https://www.reddit.com/r/${randomElement(config.subreddits)}/${config.sort}.json?t=${config.from}limit=${config.limit}`;
-    const options = {
-        uri: url,
-        json: true,
-        simple: true,
-    };
+const maxRetry = config.maxRetry
```
Comment (the-pat) [inline index.js]: Lets remove the global `maxRetry` variable since we already have access to `config.maxRetry`.

File: index.js [hunk 3]
```diff
@@ -5,38 +5,48 @@ const request = require('request');
...
 
-    const path = config.directory + '/wallpaper';
-    let name = '';
-
-    if (!fs.existsSync(config.directory)) {
-        fs.mkdirSync(config.directory);
+const change = (start) => {
```
Comment (the-pat) [inline index.js]: Give the start parameter an initial value of 0. [Check out this MDN article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters).

File: index.js [hunk 4]
```diff
@@ -5,38 +5,48 @@ const request = require('request');
...
+    if(start === maxRetry) {
+        console.log('We try our best but sorry')
```
Comment (the-pat) [inline index.js]: Let's change the string to: `'We tried our best, but were unable to change the wallpaper. Sorry.'`

File: index.js [hunk 5]
```diff
@@ -65,4 +75,4 @@ const download = (uri, path) => new Promise(resolve => request(uri).pipe(fs.crea
 
 const getType = path => new Promise((resolve, reject) => fs.readFile(path, (err, data) => err ? reject(err) : resolve(fileType(data))));
 
-change();
\ No newline at end of file
+change(0);
```
Comment (the-pat) [inline index.js]: Once you give `start` a default value, you can call `change()`.

**Review by the-pat — CHANGES_REQUESTED** [review]
Thanks for working on this project!

I have a few minor changes I'd like, then it's good to merge!

**Comment by oat9002** [comment]
I've fixed following by the comment

**Pull request merged**